### ISSUE TITLE
Add the level of the rule directly on the item listed

### ISSUE THREAD

**Issue by aguscha333** [a11yproject/a11yproject.com#1209]
Please review our Contributing guidelines:
https://a11yproject.com/contributing-guidelines/

## Summary
I think it would be useful to be able to quickly see the level of accessibility (A, AA, or AAA). Ideally, a webpage should be able to check them all off but it is also important to know what to prioritize to make the most impact sooner.

### Expected result

> Show the level (A, AA, or AAA) on each item in the accessibility checklist when the items are either collapsed or not.

### Actual result

> This is not available yet.

I would like to know if you think this is a good idea and how possible its implementation is as well. With some guidance, I'd be happy to take a crack at it.

**Comment by ericwbailey** [comment]
Hey @aguscha333, I think this is a lovely idea! Some of the checklist items represent multiple WCAG success criterion, so this might get a little tricky. Do you think it'd be an average, or the highest applicable level?

As for implementation, [here's where the checklist content is stored](https://github.com/a11yproject/a11yproject.com/blob/main/src/_data/checklists.json). We could add a `wcag` entry for the JSON data, then update [the checklist partial](https://github.com/a11yproject/a11yproject.com/blob/main/src/_includes/checklist.njk) to output it.

**Comment by aguscha333** [comment]
Hmm good question, I was not expecting them to relate to more than one level. I think if we are only showing one level I would show the lowest applicable level since it's the first level that people are going to try to cover so listing the highest one only could lead to the item being dismissed and left to do later. That being said, since there are only 3 levels I think that it could be possible to list them all like `A, AA` for example.

In case I want to contribute and add the change myself, I would need a bit of guidance in gathering the levels associated with each item in the checklist JSON. Each item has a `"wcag"` property that has the name of the guideline and then also a link. I haven't checked all of them but it would just be to open the link and copy the levels listed there?

I would add a property called `wcagLevels` and for simplicity the value could be just a string like `"A, AA"` or it could be an array like `["A", "AA"]` and then parsed some how. What do you think?

**Comment by resource11** [comment]
I can take a crack at this, @ericwbailey

**Comment by ericwbailey** [comment]
>I think if we are only showing one level I would show the lowest applicable level since it's the first level that people are going to try to cover so listing the highest one only could lead to the item being dismissed and left to do later. 

This makes sense to me!

>In case I want to contribute and add the change myself, I would need a bit of guidance in gathering the levels associated with each item in the checklist JSON.

@resource11: Is this something you'd be comfortable helping out with?

>I haven't checked all of them but it would just be to open the link and copy the levels listed there?

Sounds like a perfectly reasonable starting point to me 🙂

>I would add a property called `wcagLevels` and for simplicity the value could be just a string like `"A, AA"` or it could be an array like `["A", "AA"]` and then parsed some how. What do you think?

>That sounds like a good approach. I think a string should be sufficient for now, I don't anticipate needing an array for anything fancy down the line.

**Comment by resource11** [comment]
I'm comfortable helping out with this one, @ericwbailey!

**Comment by aguscha333** [comment]
Hey @resource11 did you get a chance to work on this? if not maybe I could take a look?
